Artifacts/windows-nvm/Artifactfile.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2016-11-28/dtlArtifacts.json",
+    "title": "NVM",
+    "publisher": "Ross Bohr",
+    "description": "Installs NVM using the Chocolatey package manager",
+    "tags": [
+        "Windows",
+        "Node",
+        "NodeJS",
+        "NVM",
+        "Javascript"
+    ],
+    "iconUri": "https://cdn.rawgit.com/ferventcoder/chocolatey-packages/02c21bebe5abb495a56747cbb9b4b5415c933fc0/icons/nodejs.png",
+    "targetOsType": "Windows",
+    "runCommand": {
+        "commandToExecute": "[concat('powershell.exe -ExecutionPolicy bypass \"& ./startChocolatey.ps1 -PackageList ', 'nvm', '\"')]"
```

Comment:
Here the command is changed from the original command to install the "nodejs" package to a command which would install the "nvm" package.